crates/agent/src/discovers/specs.rs

```diff
@@ -33,40 +35,74 @@ pub fn parse_response(
     ))
 }
 
+#[derive(Clone, Copy, Serialize, Deserialize, Debug, PartialEq)]
+pub enum BindingType {
```

Comment:
nit: this feels pretty heavy-weight as compared to anyhow::bail with `.context()`
It's not intended to be machine accessible, right?

Reply:
right. This is just to make it easier to test error cases of `merge_capture`